README.md

```diff
@@ -119,28 +119,71 @@ multiple labels can refer to the same bundle via its commit hash.
+While there's precisely one application container per Argo node,
+a Kubernetes container created from an arbitrary image,
+sidecars are additional containers in the same Kubernetes pod
+-- or Argo DAG node, we can say, approximately synonymously --
+that concert datamon-based data-ferrying setups with the application container.
+
+_Aside_: as additional kinds of data sources and sinks are added,
+we may also refer to "sidecars" as "batteries," and so on as semantic drift
+of the shell scripts shears away feature creep in the application binary.
+
+There are currently two batteries-included® images
+
+* `gcr.io/onec-co/datamon-fuse-sidecar`
+  provides hierarchical filesystem access
+* `gcr.io/onec-co/datamon-pg-sidecar`
+  provides PostgreSQL database access
+
+Both are versioned along with
 [github releases](https://github.com/oneconcern/datamon/releases/)
-of the desktop binary:  to access release `0.4` as listed on the github releases page,
-use the tag `v0.4` as in `gcr.io/onec-co/datamon-fuse-sidecar:v0.4` when
-writing Dockerfiles or Kubernetes-like YAML that accesses the sidecar container image.
+of the
+[desktop binary](#os-x-install-guide).
+to access recent releases listed on the github releases page,
+use the git tag as the Docker image tag:
+At time of writing,
+[v0.7](https://github.com/oneconcern/datamon/releases/tag/v0.7)
+is the latest release tag, and (with some elisions)
+```yaml
+spec:
+  ...
+  containers:
+    - name: datamon-sidecar
+    - image: gcr.io/onec-co/datamon-fuse-sidecar:v0.7
+  ...
+```
+would be the corresponding Kubernetes YAML to access the sidecar container image.
+
+_Aside_: historically, and in case it's necessary to roll back to an now-antient
+version of the sidecar image, releases were tagged in git without the `v` prefix,
+and Docker tags prepended `v` to the git tag.
+For instance, `0.4` is listed on the github releases page, while
+the tag `v0.4` as in `gcr.io/onec-co/datamon-fuse-sidecar:v0.4` was used when writing
+Dockerfiles or Kubernetes-like YAML to accesses the sidecar container image.
 
 Users need only place the `wrap_application.sh` script located in the root directory
-of the sidecar container within the main container.  This can be accomplished via
-an `initContainer` without duplicating version of the Datamon sidecar image in
-both the main application Dockerfile as well as the YAML.  When using a block-storage GCS
-product, we might've specified a data-science application's Argo DAG node with something
-like
+of each of the sidecar containers within the main container.
+This
+[can be accomplished](https://github.com/oneconcern/datamon/blob/master/hack/k8s/example-coord.template.yaml#L15-L24)
+via an `initContainer` without duplicating version of the Datamon sidecar
+image in both the main application Dockerfile as well as the YAML.
+When using a block-storage GCS product, we might've specified a data-science application's
+Argo DAG node with something like
```

Comment:
This part is most interesting. I believe we could use an example here (or with a link), demonstrating the typical workflow of a user.

Reply:
:/ for further consideration at office today..